.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: ruby
+rvm: 2.4.1
+git:
+  depth: 1
+before_install:
+  - sudo add-apt-repository -y ppa:hyperair/zsh-backports # Trusty's zsh deos not support pipefail
```

Comment:
```suggestion
  - sudo add-apt-repository -y ppa:hyperair/zsh-backports # Trusty's zsh does not support pipefail
```

Reply:
Doesn't that mean that our script should not rely on pipefail?

Reply:
No - we're adding pipefail only for the testing to ensure that our script is resilient to users using that option.